rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/activity/CovidPatientRegisterActivityTest.java

```diff
@@ -72,15 +77,15 @@ public void testLanguageSwitcherDialogShouldSetCorrectLocale() throws Exception
 
     private void verifyLocaleIsSaved(ListView listView, int position, String locale) {
         listView.performItemClick(null, position, listView.getItemIdAtPosition(position));
-        Assert.assertEquals(locale, LangUtils.getLanguage(RDTApplication.getInstance()));
-        Assert.assertEquals(locale, RDTApplication.getInstance().getResources().getConfiguration().locale.getLanguage());
+        Assert.assertEquals(locale, new Locale(LangUtils.getLanguage(covidPatientRegisterActivity)).getLanguage());
```

Comment:
`Assert.assertEquals(locale, RDTApplication.getInstance().getResources().getConfiguration().locale.getLanguage());` is also a valid check, please revert it.

Reply:
Also, shouldn't this be `Assert.assertEquals(locale, LangUtils.getLanguage(covidPatientRegisterActivity));`?

Reply:
if I will revert this method so I will need to call the `Utils.updateLocale` method. https://github.com/onaio/rdt-standard/pull/579#discussion_r509976237

Reply:
I don't think that will be necessary. You should be able to simply revert the changes and the test should pass since what was there before is the expected behaviour.

Reply:
changes reverted